doc/orchestrator.rst

```diff
@@ -165,3 +165,399 @@ A full example of configuring KeyDB to run in SmartSim is shown below
   export REDIS_CONF=/path/to/keydb.conf
 
   # run smartsim workload
+
+Multiple Orchestrators
+======================
+
+SmartSim offers functionality to automate the deployment of multiple
+databases on an HPC cluster, supporting workloads that require multiple
+``Orchestrators`` for a ``Experiment``. For instance, a workload may consist of a
+simulation with high inference performance demands, along with an analysis and
+visualization workflow connected to the simulation. In such cases, SmartSim
+allows you to augment the number of clustered orchestrators or provision multiple
```

Comment:
Delete the sentence "In such cases...the workload can scale to meet its demands." It's a little redundant